activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -102,6 +102,7 @@ def dbconsole(config, options = {})
         binary:       { name: "blob" },
         boolean:      { name: "boolean" },
         json:         { name: "json" },
+        enum:         { name: "varchar" },
```

Comment:
I don't understand how this works. I get that when running a migration with an enum, it will just create a `varchar` field, but then when dumping the schema, how do you know that `varchar` is a `t.enum` and no `t.string`?